content/en/threat-model.md

```diff
@@ -0,0 +1,141 @@
+|                        |                  | x             |                       |              |               |              |             | Can remove certificates (invalidate them) by indicating that they are not in the log when Bob queries, fork attack by showing different times or certificates to different users (for things Fulcio has signed), and may block the issuance of Fulcio certificates by not creating an SCT.  | Deletion / fork may be detected by Fulcio CT log monitors. |
+|                        |                  |               |                       | x            |               |              |             | Can change the timestamp on Fulcio signatures used outside the validity window to make them appear valid when Bob queries.  Can deny the existence of valid signatures leading to them being untrusted.  | Can later recover using offline delegation in Sigstore’s TUF root of trust.  Deletion or fork attacks may be detected by Rekor monitors. Changes to the Rekor CT log would be challenging to recover as it eliminates the append-only promise of Rekor.                                                                                         |
+|                        |                  | x             | x                     |              |               |              |             | Same as Fulcio CT log | Replay and fork attacks are undetected.                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|                        |                  | x             |                       | x            |               |              |             | Same as Fulcio CT log + Rekor server, but can now use the replayed certs created by Fulcio CT log because Rekor will not report that they are already in the log, even with more sophisticated monitors.                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
I would avoid abbreviations as there aren't any others.

```suggestion
|                        |                  | x             |                       | x            |               |              |             | Same as Fulcio CT log and Rekor server, but can now use the replayed certs created by Fulcio CT log because Rekor will not report that they are already in the log, even with more sophisticated monitors.                                                                                                                                                                                                                                                                                                                                                     |
```